How to use under ArchLinux from issue#47

Sandman6z opened the pull request:
Thanks to the author's invitation, I made a modification to the environment configuration part of the readme and proposed a PR for issue #47.

kisvegabor commented on README.md:
```diff
@@ -35,6 +40,13 @@ sudo apt-get update && sudo apt-get install -y build-essential libsdl2-dev cmake
 ![image](https://github.com/lvgl/lv_port_pc_vscode/assets/7599318/f527b235-5718-4949-b5f0-bd807b3a64ba)
 4. Click the Play button or hit F5 to start debugging.
+   
+* ***ArchLinux User***  
```
Simple bold?
```suggestion
**ArchLinux User**  
```

kisvegabor reviewed (commented):
Thank you! :tada: 

Just a minor formatting note.

Sandman6z commented:
> Thank you! 🎉
> 
> Just a minor formatting note.

---

Sorry! 😢
This is the first time I have modified the issue, I have modified the suggestion, if you can't submit it successfully, please contact me in time.
Thank you!

kisvegabor reviewed (approved):
Looks good! Thank you! Especially for having your first contribution! 🎉
